src/client/datascience/data-viewing/jupyterVariableDataProvider.ts

```diff
@@ -47,10 +49,14 @@ export class JupyterVariableDataProvider implements IJupyterVariableDataProvider
                 default:
                     normalizedType = ColumnType.String;
             }
-            return {
+            const col = {
                 key: column.key,
                 type: normalizedType
-            };
+            } as { key: string; type: ColumnType; describe?: string };
+            if (column.describe) {
+                col['describe'] = column.describe;
```

Comment:
What happens if this column isn't here? Do you need a default value for this?

Reply:
Not exactly sure what you meant. Do you mean if describe doesn't exist? If that's the case, it can be undefined, I could also make it the empty string

Reply:
Yeah ```if (column.describe)``` is undefined, so ```col['describe']``` never gets created. Will that break something else later?

Reply:
I think it's okay because it is potentially undefined from the type definition. I can play it safe and give it an empty string though.